README.md

```diff
@@ -2,80 +2,183 @@ Airport Challenge
 =================
 
 ```
-         ______
-        __\____\___
+	   ______
+	  __\____\___
 =  = ==(____DFA____)
-           \_____\__________________,-~~~~~~~`-.._
-          /     o o o o o o o o o o o o o o o o  |\_
-          `~-.__       __..----..__                  )
-                `---~~\___________/------------`````
-                =  ===(_________)
+		 \_____\__________________,-~~~~~~~`-.._
+		/     o o o o o o o o o o o o o o o o  |\_
+		`~-.__       __..----..__      Raj         )
+			`---~~\___________/------------````````
+		=	=  ===(_________)
 
 ```
 
-Instructions
+Installation Steps:
 ---------
 
-* Feel free to use google, your notes, books, etc. but work on your own.
-* Keep it SIMPLE - it's not nearly as complicated as it first may look.
-* You must [submit your challenge](https://airtable.com/shrUGm2T8TYCFAmjN) by the deadline, wherever you get to.
-* Use your own test framework and evidence your test-driven development by committing on passing tests.
-* Please write your own README detailing how to install your project, how to run the tests, how you approached the problem and provide screenshots of interacting with your program.
-* If you refer to the solution of another coach or student, please put a link to that in your README.
-* Please create separate files for every class, module, and spec.
+1. Clone the repository to the local machine.
+2. Make sure the current working directory in the terminal is './airport-challenge'.
+3. Run command 'npm install' to automatically install all project dependecies using the package.json.
+4. To run tests the following commands can be used:
+    * 'node specRunner.js'
+    * 'npm test'
 
```

Comment:
Installation steps could have been written in a code block for better reading and separation.